components/reusable-components/footer/Footer.tsx

```diff
@@ -32,13 +39,24 @@ const Footer: React.FC = () => {
   // eslint-disable-next-line no-unused-vars
   const [errorMessage, setErrorMessage] = useState<boolean>(false);
 
-  // eslint-disable-next-line no-unused-vars
+  const [turnstileToken, setTurnstileToken] = useState<string | null>(null);
+
   const handleSubmit = async (
     values: { name: string; email: string },
     { resetForm }: FormikHelpers<{ name: string; email: string }>
   ) => {
+    if (!turnstileToken) {
+      toast.error('Please complete the captcha');
+      return;
+    }
```

Comment:
@Mikivitkovski 
How will the user know what to do in this scenario when the <Turnstile/> is invisible?

cc: @RevanchistX

Reply:
Maybe we can reload the page if there is no token?

Reply:
@edichoska will update this in a future scenario, we are going to test the staging environment and the keys for now.

Reply:
> Maybe we can reload the page if there is no token?

Reloading the page might lead to infinite reloading if there's something misconfigured. 
The only thing the end user can do is contact the page administrator, which ironically wouldn't work on the contact form.
We can write something like "I guess someone didn't learn how to validate forms hehe" error, if we have to.

Reply:
we should use resetWidget() in this scenario, so the user can try again.
but sure, lets test first.

Reply:
@edichoska We tested on Staging and it works as intended.